webapp/src/components/modals/create_issue/index.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {getPost} from 'mattermost-redux/selectors/entities/posts';
+
+import {id as pluginId} from 'manifest';
+import {closeCreateIssueModal, createIssue} from 'actions';
+
+import CreateIssueModal from './create_issue';
+
+const mapStateToProps = (state) => {
+    const postId = state[`plugins-${pluginId}`].createIssueModalForPostId;
+    const post = getPost(state, postId);
```

Comment:
I suggest we have this component receive the `post` (or even just `postMessage` if we don't need anything else) so we can avoid a double fetch to the API to get the post.

Reply:
I am not sure what to do here. Might be better if you can write up the fix.

Reply:
@jfrerich what about this one? I think we can live with the double-fetch since it's a user-initiated transaction and does not present a scalability challenge, but we should at least file this as a ticket to eliminate the double-fetch if it's not fixed yet,

Reply:
@lev.  Let's leave the double fetch and I'll create a HW ticket.

Reply:
Ticket created. https://github.com/mattermost/mattermost-plugin-github/issues/206